osc-server/src/test/java/org/osc/core/broker/service/servicefunctionchain/UpdateServiceFunctionChainServiceTest.java

```diff
@@ -82,14 +84,67 @@ public void testDispatch_WhenVsIdsListUpdateValidationSuccessful_ValidationSucce
 		request.setDto(dto);
 		request.setName(this.sfc.getName());
 		List<Long> vsIds = new ArrayList<Long>();
+		vsIds.add(this.vs.getId());
+		vsIds.add(this.vs1.getId());
+		request.setVirtualSystemIds(vsIds);
+		Mockito.when(this.service.validator.validateAndLoad(request)).thenReturn(this.sfc);
+		// Act.
+		BaseJobResponse response = this.service.dispatch(request);
+		Assert.assertNotNull("The returned response should not be null.", response);
+	}
+	
+	@Test
+	public void testDispatch_sfcVirutalSystemIdsInOrder_ValidationSucceeds() throws Exception {
+		
+		BaseDto dto = new BaseDto();
+		dto.setId(this.sfc.getId());
+		dto.setParentId(this.vc.getId());
+		request.setDto(dto);
+		request.setName(this.sfc.getName());
+		List<Long> vsIds = new ArrayList<Long>();
 		vsIds.add(this.vs1.getId());
 		vsIds.add(this.vs.getId());
 		request.setVirtualSystemIds(vsIds);
 		Mockito.when(this.service.validator.validateAndLoad(request)).thenReturn(this.sfc);
 		// Act.
+		//update sfc with virtual system ids list {2,1}
```

Comment:
// Assert. The AAA comments really help to review and keep the longer tests like this organized.